.github/workflows/build_and_test.yml

```diff
@@ -129,6 +129,7 @@ jobs:
     name: "Build docs"
     uses: ./.github/workflows/_docs.yml
     needs: build-torch-xla
+    if: github.event_name == 'push'
```

Comment:
Does this mean push to the `master` branch by default?

Reply:
Thats right